lib/supply/client.rb

```diff
@@ -292,10 +292,34 @@ def clear_screenshots(image_type: nil, language: nil)
       raise result.error_message if result.error?
     end
 
+    def upload_obb(obb_file_path)
+      result = api_client.execute(
+        api_method: @android_publisher.edits.expansionfiles.upload,
+        parameters: {
+          packageName: current_package_name,
+          editId: current_edit.data.id,
+          expansionFileType: obb_main?(obb_file_path),
+          media: {
+            mimeType: 'application/octet-stream',
+            body: obb_file_path.read
```

Comment:
SUre that works ? We don't need to use a `Google::APIClient::UploadIO.new(File.expand_path(path_to_obb), 'application/vnd.android.obb')`